components/camel-quartz/src/test/java/org/apache/camel/routepolicy/quartz/SpringMultiplePoliciesOnRouteTest.java

```diff
@@ -34,7 +34,6 @@ public void testMultiplePoliciesOnRoute() throws Exception {
 
         for (int i = 0; i < size; i++) {
             template.sendBody(url, "Message " + i);
-            Thread.sleep(3);
         }
```

Comment:
I think we need this `Thread.sleep` if we want to throttle sending the messages. I have seen this pattern elsewhere. I don't think you need to fix this one right now (see: https://issues.apache.org/jira/browse/CAMEL-19986)